grunt_tasks/plugin.js

```diff
@@ -187,6 +184,18 @@ module.exports = function (grunt) {
                 config = grunt.file.readYAML(yml);
             }
 
+            var doAutoConfig = (
+                typeof config.grunt !== "object" ||
+                config.grunt.autoconf === (void 0) ||
```

Comment:
Just a habit of mine from when you used to be able to redefine `undefined`.  I just checked on chrome and node.js and it looks like it's not necessary, anymore?